rapids-cmake/test/detail/generate_installed_CTestTestfile.cmake

```diff
@@ -224,7 +228,7 @@ function(determine_install_location_of_all_targets)
         # Continue to add the lines of `file(INSTALL` till we hit the closing `)` That allows us to
         # support multiple line file commands
         string(APPEND command_contents "${line}")
-        if(line MATCHES "\\)")
+        if(line MATCHES "\\)$")
           # We have all the lines for this file command, now parse it
           extract_install_info(${command_contents})
```

Comment:
This needs to be quoted now, otherwise it will be broken up by semicolons and `line` will only get the first part.

Reply:
This works as expected:
```cmake
set(line "a;b;c;)")
message(STATUS "line: ${line}")
if(line MATCHES "\\)$")
  message(STATUS "line ends with )")
endif()
```
Generates output:
```
-- line: a;b;c;)
-- line ends with )
```

Your concern about semi-colons is valid, as the `extract_install_info` does need to be changed to use `ARGN` to handle the fact that any semicolons would make the function be passed 2+ arguments, and not the 1 we are presuming